src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/legaladvisor/DocAssemblyTemplateBodyMapper.java

```diff
@@ -101,4 +102,23 @@ public DocAssemblyTemplateBody from(CCDCase ccdCase, UserDetails userDetails) {
                 ccdCase.getExpertReportPermissionPartyGivenToDefendant() == YES)
             .build();
     }
+
+    public DocAssemblyTemplateBody generalLetterBody(CCDCase ccdCase, UserDetails userDetails) {
+        LocalDate currentDate = LocalDate.now(clock.withZone(UTC_ZONE));
+        return DocAssemblyTemplateBody.builder()
+            .currentDate(currentDate)
+            .referenceNumber(ccdCase.getPreviousServiceCaseReference())
+            .caseworkerName(userDetails.getFullName())
+            .caseName(ccdCase.getCaseName())
+            .partyName(ccdCase.getIssueLetterContact() == CCDContactPartyType.CLAIMANT
```

Comment:
Determining the party to be done first, so the following 10 statements can be simplified